Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -255,6 +255,10 @@ void AuthenticatorCoordinator::discoverFromExternalSource(const Document& docume
     // Only FIDO AppID Extension is supported.
     if (options.extensions && !options.extensions->appid.isNull()) {
         // The following implements https://www.w3.org/TR/webauthn/#sctn-appid-extension as of 4 March 2019.
+        if (options.extensions->appid.isEmpty()) {
```

Comment:
This covers whenever the parameter isn't passed at all. You want `!isNull && isEmpty` to only cover when it's passed, but is the empty string.